### PR TITLE
[examples] Turnip Town V2

### PR DESCRIPTION
**Pull request by damirka**
## Description 

New and shiny Turnip Town.

## Test plan 

How did you test the new or updated feature?

---

## Release notes

Check each box that your changes affect. If none of the boxes relate to your changes, release notes aren't required.

For each box you select, include information after the relevant heading that describes the impact of your changes that a user might notice and any actions they must take to implement updates. 

- [ ] Protocol: 
- [ ] Nodes (Validators and Full nodes): 
- [ ] Indexer: 
- [ ] JSON-RPC: 
- [ ] GraphQL: 
- [ ] CLI: 
- [ ] Rust SDK:
- [ ] REST API:

### CODE REVIEW

File: examples/turnip-town/move/tests/turnip_tests.move [hunk 1]
```diff
@@ -1,85 +1,85 @@
...
+    assert!(turnip.size() == 150);
+    assert!(turnip.freshness() == 8_75);
+    assert!(w == 610);
+
+    // Growth while in drought.
+    // Day 1: fresh: 4_37
+    // Day 2: fresh: 2_18
+    // Day 3: fresh: 1_09
+    // Day 4: fresh:   54
+    // Day 4: fresh:   27
```
Comment (amnn) [inline examples/turnip-town/move/tests/turnip_tests.move]: (This is my typo, but just noting it so I don't forget).

```suggestion
    // Day 5: fresh:   27
```

File: examples/turnip-town/move/tests/royalty_policy_tests.move [hunk 2]
```diff
@@ -1,86 +1,84 @@
...
-        abort 0
-    }
+module turnip_town::royalty_policy_tests;
+
+use sui::coin;
+use sui::sui::SUI;
+use sui::test_scenario as ts;
+use sui::transfer_policy as policy;
+use sui::transfer_policy_tests as test;
+use kiosk::royalty_rule as royalty;
```
Comment (amnn) [inline examples/turnip-town/move/tests/royalty_policy_tests.move]: We can't have a dependency on `MystenLabs/apps` from `MystenLabs/sui` -- that's why `turnip_town` has its own `royalty_policy` -- can you add it back please?

**Review by amnn — COMMENTED** [review]
The formatting changes all look great, thanks for that!

We can't depend on `MystenLabs/apps` though -- because the example is going to be run at diff-time in CI and we want to avoid taking a dependency on an external repo. That's why the example defines its own royalty policy.

**Comment by damirka** [comment]
@amnn I've reverted royalty changes.

**Review by amnn — APPROVED** [review]
Thank you!

**Pull request merged**